rdr_service/offline.yaml

```diff
@@ -4,12 +4,10 @@ runtime: python37
 service: offline
 entrypoint: gunicorn -c rdr_service/services/gunicorn_config.py --timeout 18000 --max-requests 10 --max-requests-jitter 3 rdr_service.offline.main:app
 
-# Default AppEngine configuration (B1) has only 128 MB of RAM and 600 MhZ CPU, which makes
-# the metrics pipeline run very slowly. Bumping up to 1 GB of RAM and 2.4 GHz to speed things up.
-# https://cloud.google.com/appengine/docs/standard/
-instance_class: B4
-# We need to specify basic scaling in order to use a backend instance class.
-basic_scaling:
-  max_instances: 10
-  idle_timeout: 1m
+# Because we are changing to an auto-scaled class, need to specify automatic_scaling
+# https://cloud.google.com/appengine/docs/standard/reference/app-yaml?tab=python#scaling_elements
+instance_class: F4_1G
+automatic_scaling:
+  min_idle_instances: automatic
```

Comment:
We're covered on the "Warning" in the documentation related to `min_idle_instances` because we have `inbound_services` warmup configured for all of our services (part of the base config), is that correct?